### ISSUE TITLE
Space panel avatar colour and alignment regressions

### ISSUE THREAD

**Issue by weeman1337** [vector-im/element-web#24369]
### Steps to reproduce

1. Enable light mode
2. Have some spaces without image
3. Look at the space panel

#### What did you expect?

Dark grey letters on a light grey background

#### What happened instead?

Dark grey letters on coloured background. Hard to read

![image](https://user-images.githubusercontent.com/6216686/215510365-594a1bf5-ff55-4f48-81fc-c828c1b68340.png)

### Operating system

Ubuntu 22.04.1 LTS

### Browser information

Firefox 109.0.1

### URL for webapp

https://develop.element.io/

### Application version

Element version: 50f2b532e907-react-3e2bf5640e17-js-c142232f4d13 Olm version: 3.2.12

### Homeserver

matrix.org

### Will you send logs?

No

**Comment by robintown** [comment]
(They're supposed to be dark gray letters on a light gray background)

**Comment by weeman1337** [comment]
> (They're supposed to be dark gray letters on a light gray background)

upsi -  I have updated the description. Thanks

**Comment by SatyarajRana** [comment]
@weeman1337 There are three options i think will look good, please tell which one to implement:
1. The one you proposed
2. Black letters on coloured backgrounds
3. Darker shade of a color for letters and lighter shade of same color for background

**Comment by weeman1337** [comment]
@SatyarajRana this is a regression. The current stable version (for instance deployed here https://app.element.io/ ) is working correctly 

![image](https://user-images.githubusercontent.com/6216686/215519428-4d71647e-52bc-4787-a371-a07860caf9f6.png)

If you want to work on it, you should find the change that is causing the problem and fix it accordingly.

**Comment by SatyarajRana** [comment]
Oh, got it

**Comment by t3chguy** [comment]
Related space panel regression around alignment:

Staging
![image](https://user-images.githubusercontent.com/2403652/215552681-49e62f69-6f23-4697-b175-2a085e5c6bd2.png)

Develop
![image](https://user-images.githubusercontent.com/2403652/215552707-53e32f8d-7893-4ab6-b318-0addcb87b7e1.png)

The margin below user avatar increased in size, knocking the 2 hairlines out of line

**Comment by t3chguy** [comment]
Regressed by https://github.com/matrix-org/matrix-react-sdk/commit/a8aa4de4b4c14a6a995071aab3f2f871b3ff2388
